htdocs/commande/list.php

```diff
@@ -553,6 +559,11 @@
 				}
 			}
 
+			if ($currentIndex <= (getDolGlobalInt("MAXREFONDOC") ? getDolGlobalInt("MAXREFONDOC") : 10)) {
+				$objecttmp->note_public = dol_concatdesc($objecttmp->note_public, $langs->transnoentities($cmd->ref).(empty($cmd->ref_client) ? '' : ' ('.$cmd->ref_client.')'));
+				$objecttmp->update($user);
+			}
+			
```

Comment:
To avoid such errors, you should think to activate the "pre-commit" hooks in your development environment:
See https://wiki.dolibarr.org/index.php?title=Environment_and_development_tools#Install_Pre-commit_githook